### PR TITLE
Feature/opt in booking

### PR DESCRIPTION
**Pull request by Malte-D**
EventTypes can now be opt-in, which requires a manual confirmation (or rejection) by the booked user. Reminder emails will also be sent to the user.

### CODE REVIEW

File: pages/api/cron/bookingReminder.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```
Comment (femyeda) [inline pages/api/cron/bookingReminder.ts:1]: Not sure how this works. Vercel doesn't support cron jobs natively. 

Secondly req.query.apiKey equals process.env.CRON_API_KEY checks what exactly?

Comment (femyeda) [reply pages/api/cron/bookingReminder.ts:1]: This comment is about the entire file in general, not just the import.

**Review by femyeda — APPROVED** [review]

File: pages/api/cron/bookingReminder.ts (hunk 1 above)
Comment (Malte-D) [reply pages/api/cron/bookingReminder.ts:1]: It is just a normal endpoint, that is called by an external scheduler

Comment (femyeda) [reply pages/api/cron/bookingReminder.ts:1]: Ahh okay, who's providing that?

**Pull request merged**